rdfhash/utils/__init__.py

```diff
@@ -2,6 +2,37 @@
 
 from .hash import hash_types
 
+triple_mime_types = {
+    "text/turtle",
+    "application/n-triples",
+}
+
+rdf_ext_to_mime_types = {
+    "ttl": "text/turtle",
+    "nt": "application/n-triples",
+    "trig": "application/trig",
+    "nq": "application/n-quads",
+    "rdf": "application/rdf+xml",
+}
+rdf_mime_types = {
```

Comment:
This is just `rdf_ext_to_mime_types.values()`, there is no need to maintain a separate list.